custom_components/alexa_media/manifest.json

```diff
@@ -8,6 +8,6 @@
   "iot_class": "cloud_polling",
   "issue_tracker": "https://github.com/alandtse/alexa_media_player/issues",
   "loggers": ["alexapy", "authcaptureproxy"],
-  "requirements": ["alexapy==1.29.8", "packaging>=20.3", "wrapt>=1.14.0"],
-  "version": "5.7.10"
+  "requirements": ["alexapy==1.29.9", "packaging>=20.3", "wrapt>=1.14.0"],
```

Comment:
Only bump alexapy. I also believe alexapy is in other files. Please find those too.